Documentation/network/servicemesh/gateway-api/installation.rst

```diff
@@ -31,6 +31,17 @@ Prerequisites
   so your environment will need to support this. Alternatively, since Cilium 1.16+,
   you can directly expose the Cilium L7 proxy on the :ref:`host network <gs_gateway_host_network_mode>`.
 
+Known Issues
```

Comment:
Would love to have this in the separate file and then we can just add link/reference for both Ingress and Gateway API. 

Another point is to add the github issues with below syntax.

`GH#31653 <https://github.com/cilium/cilium/issues/31653>` -> for the bpf.masquerade=true issue

`GH#24318 <https://github.com/cilium/cilium/issues/24318>` -> for multiple device issue

Reply:
Great, thanks. Done!